weave/trace/op.py

```diff
@@ -350,7 +356,7 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
             wrapper.calls = partial(calls, wrapper)  # type: ignore
 
             wrapper.__call__ = wrapper  # type: ignore
-            wrapper.__self__ = wrapper  # type: ignore
+            wrapper.__self__ = None  # type: ignore
```

Comment:
`__self__` was completely unused before - coopting it to mean "the self argument for ops that are gotten"